src/Germany/Mews.Fiscalizations.Germany.Tests/V2/TestFixture.cs

```diff
@@ -1,21 +1,54 @@
-﻿using Mews.Fiscalizations.Germany.V2;
+﻿using FuncSharp;
+using Mews.Fiscalizations.Germany.V2;
 using Mews.Fiscalizations.Germany.V2.Model;
 using System;
+using System.Threading.Tasks;
 
 namespace Mews.Fiscalizations.Germany.Tests.V2
 {
     public static class TestFixture
     {
-        public static readonly Guid ClientId = new Guid(Environment.GetEnvironmentVariable("german_client_Id") ?? "INSERT_CLIENT_ID");
-        public static readonly Guid TssId = new Guid(Environment.GetEnvironmentVariable("german_tss_id") ?? "INSERT_TSS_ID");
         public static readonly ApiKey ApiKey = ApiKey.Create(Environment.GetEnvironmentVariable("german_api_key") ?? "INSERT_API_KEY").Success.Get();
         public static readonly ApiSecret ApiSecret = ApiSecret.Create(Environment.GetEnvironmentVariable("german_api_secret") ?? "INSERT_API_SECRET").Success.Get();
         public static readonly string AdminPin = Environment.GetEnvironmentVariable("german_admin_pin") ?? "INSERT_ADMIN_PIN";
-        public static readonly string AdminPuk = Environment.GetEnvironmentVariable("german_admin_puk") ?? "INSERT_ADMIN_PUK";
 
-        public static FiskalyClient GetFiskalyClient()
+        // Fiskaly deletes the test environment data each week, so we should recreate the deleted data for tests.
```

Comment:
This sentence makes no sense. We're deleting them because of the quota on how many can exist.